cmd/solvercli/cmd/settle_orders.go

```diff
@@ -0,0 +1,153 @@
+package cmd
+
+import (
+	"fmt"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/skip-mev/go-fast-solver/hyperlane"
+	"github.com/skip-mev/go-fast-solver/shared/clients/coingecko"
+	"github.com/skip-mev/go-fast-solver/shared/clients/utils"
+	"github.com/skip-mev/go-fast-solver/shared/evmrpc"
+	"github.com/skip-mev/go-fast-solver/shared/txexecutor/evm"
+
+	"github.com/skip-mev/go-fast-solver/db/connect"
+	"github.com/skip-mev/go-fast-solver/db/gen/db"
+	"github.com/skip-mev/go-fast-solver/ordersettler"
+	"github.com/skip-mev/go-fast-solver/shared/clientmanager"
+	"github.com/skip-mev/go-fast-solver/shared/config"
+	"github.com/skip-mev/go-fast-solver/shared/keys"
+	"github.com/skip-mev/go-fast-solver/shared/lmt"
+	"github.com/skip-mev/go-fast-solver/shared/txexecutor/cosmos"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+	"golang.org/x/net/context"
+)
+
+var settleCmd = &cobra.Command{
+	Use:   "settle-orders",
+	Short: "Settle pending order batches",
+	Long: `Settle all pending order batches immediately without any threshold checks (ignoring configured BatchUUSDCSettleUpThreshold).
```

Comment:
The way this command is currently written, the UX involves first having run the solver to populate the db with pending settlements. Probably worth mentioning in the description

Reply:
i removed the db dependency to instead get pending settlements from the contract